.env

```diff
@@ -12,25 +12,27 @@ REACT_APP_REWARD_SERVICE_API=https://reward.stg.kyberengineering.io/api/v1
 REACT_APP_PRICE_CHART_API=https://price-chart.kyberswap.com/api
 REACT_APP_PRICE_API=https://price.stg.kyberengineering.io
 
-REACT_APP_FIREBASE_API_KEY=AIzaSyA1K_JAB8h0NIvjtFLHvZhfkFjW4Bls0bw
-REACT_APP_FIREBASE_AUTH_DOMAIN=notification---production.firebaseapp.com
-REACT_APP_FIREBASE_PROJECT_ID=notification---production
-REACT_APP_FIREBASE_STORAGE_BUCKET=notification---production.appspot.com
-REACT_APP_FIREBASE_MESSAGING_SENDER_ID=541963997326
-REACT_APP_FIREBASE_APP_ID=1:541963997326:web:a958e2bb7cbeea932679df
-REACT_APP_FIREBASE_VAPID_KEY=BGO4YGVpmvxoMlqILCfVUVfyA0M5ohAKp_0-p1jqfGm15evDld0Zrp_pTFdYFydMRW_r8wyJr162Y9BGgkp11tE
```

Comment:
remove on `.env.dev`, `.env.production`, `.env.stg` also

Reply:
updated dev and stg, production will not.

Reply:
can you help explain?

Reply:
production will use the current firebase config, no need to change

Reply:
You might have a mistake here. Current production will still use this key. But new production after your code has been merged will not.

Reply:
`But new production after your code has been merged will not.`
No, nothing changed, we will use that config.

Reply:
Can you explain why nothing changed since you have removed them in this PR?

Reply:
.env I don't remove. I replaced. and production no need to replace